src/components/ReactPlayer/CustomBarControls.scss

```diff
@@ -69,12 +93,37 @@ $controlSize: 64px;
                     outline: none;
                 }
             }
+
+            &_with-uikit-play-pause-button {
+                width: 42px;
+                height: 42px;
+                border-radius: 50%;
+                background: var(--g-color-base-background);
+                box-shadow: 0 4px 24px 0 var(--g-color-sfx-shadow-light),
+                    0 2px 8px 0 var(--g-color-sfx-shadow-light);
+
+                &:focus {
+                    outline: 2px solid var(--g-color-line-misc);
```

Comment:
maybe create mixin for outlines if it would be the same for elements?

Reply:
yes, I will create this mixin here https://github.com/gravity-ui/page-constructor/pull/576